webapp/channels/src/components/admin_console/license_settings/trial_license_card/trial_license_card.tsx

```diff
@@ -41,7 +42,7 @@ const TrialLicenseCard: React.FC<Props> = ({license}: Props) => {
         return (
             <FormattedMessage
                 id='admin.license.trialLicenseCard.expiringAfterFewDays'
-                defaultMessage='Your free trial will expire in <b>{daysCount} {daysCount, plural, one {day} other {days}}</b>. Visit our customer portal to purchase a license now to continue using Mattermost Professional and Enterprise features after trial ends.'
+                defaultMessage='Your free trial will expire in <b>{daysCount} {daysCount, plural, one {day} other {days}}</b>. Visit our customer portal to purchase a license now to continue using Enterprise Advanced features after trial ends.'
```

Comment:
Is this right? Are we directing to the customer portal still? Or should this be contact sales

Reply:
Fixed, I had missed the default message and had only done the change in the translation file.